src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Circle");
+        System.out.println("Color: " + getColor());
+        System.out.println("Radius: " + radius);
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
```

Comment:
Consider defining the value of PI as a constant. While Math.PI is a well-known constant, for other cases where magic numbers are used, it's better to define them as constants for readability and maintainability of the code.